olm-catalog/serverless-operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.3"
```

Comment:
Given that this bundle will be published to all OCP clusters, if this bundle gets added last it will set the default channel to be 4.3. Is this desired? Is there another channel that should be default in 4.5 or 4.6? Just double checking

Reply:
Shawn had always mentioned the 4.3 channel should be set as the default as long as we support 4.3 in any of our active releases.